e2e/tests/e2e.rs

```diff
@@ -29,7 +42,7 @@ fn e2e_channel_success(channel: &'static str, arch: &'static str) {
     let expected: cincinnati::plugins::internal::versioned_graph::VersionedGraph =
```

Comment:
We're importing two versions of the graphs for 4.7/4.8, but testing just the stable-4.2/4.3